closure/goog/html/trustedresourceurl.js

```diff
@@ -186,25 +186,29 @@ goog.html.TrustedResourceUrl.unwrap = function(trustedResourceUrl) {
  * Values of other types are cast to string and encoded with
  * encodeURIComponent.
  *
- * %{<label>} markers are used in the format string to indicate locations
- * to be interpolated with the valued mapped to the given label. <label>
+ * `%{<label>}` markers are used in the format string to indicate locations
+ * to be interpolated with the valued mapped to the given label. `<label>`
  * must contain only alphanumeric and '_' characters.
  *
  * The format string must start with one of the following:
- * - https://<origin>/<pathStart>
- * - //<origin>/<pathStart>
- * - /<pathStart>
+ * - `https://<origin>/<pathStart>`
+ * - `//<origin>/<pathStart>`
+ * - `/<pathStart>`
  *
- * <origin> must contain only alphanumeric, '-', .', ':', '[', and ']'.
- * <pathStart> must contain only alphanumeric, '_', '~', and '-'. If other
- * characters follow it, it must end with '/', '#' or '?'.
+ * `<origin>` must contain only alphanumeric, {@code '-', '.', ':', '['}, and
```

Comment:
I'm not convinced it makes the most sense to have the `@code` around the apostrophies, though I'm also not convinced it's a win to do it any other way... Maybe something like

`<origin>` must contain only alphanumeric or any of the following: `-.:[]`
`<pathStart>` must contain only alphanumeric or any of the following: `_~-`